rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientRegisterActivity.java

```diff
@@ -49,4 +70,55 @@ protected Class getLoginPage() {
     protected PatientRegisterActivityPresenter createPatientRegisterActivityPresenter() {
         return new CovidPatientRegisterActivityPresenter(this);
     }
+
+    private Map<String, Locale> locales = new HashMap<>();
+    private String[] languages;
+    private int currentLanguageIndex = 0;
+
```

Comment:
Global variables should be at the top of the file.

Reply:
done.